frontend/packages/component-library/src/cms/iframeEmbedContainer/stories/IframeEmbedContainer.story.tsx

```diff
@@ -0,0 +1,42 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {within, expect} from '@storybook/test';
+
+import IframeEmbedContainer, {IframeEmbedContainerProps} from '..';
+
+export default {
+  title: 'CMS/IframeEmbedContainer',
+  component: IframeEmbedContainer,
+  render: args => (
+    <div style={{height: '400px'}}>
+      <IframeEmbedContainer {...args} />
+    </div>
+  ),
+} as Meta<IframeEmbedContainerProps>;
+type Story = StoryObj<typeof IframeEmbedContainer>;
+
+const defaultArgs: IframeEmbedContainerProps = {
+  title: 'Iframe Title',
+  src: 'https://go.pardot.com/l/153401/2023-01-10/pqf2sr',
```

Comment:
To avoid eyes tests from failing flakily due to Pardot, I'd recommend we use an internal-only src.

```suggestion
  src: 'data:text/html,<h1>Hello, world!</h1>'
```

![image](https://github.com/user-attachments/assets/fd1845c7-02bc-4575-960b-aad4e0977290)

Reply:
Totally agree; using an external link for unit testing is also a bad practice.